packages/Algernon.package/ALGCore.class/instance/handleListenEvent..st

```diff
@@ -1,7 +1,13 @@
 events-processing
 handleListenEvent: anEvent
-	| isActivationString shouldToggle |
+	| isActivationString shouldToggle timestamp difference |
+	timestamp := TimeStamp current time asNanoSeconds.
+	difference := timestamp - (lastHalfActivationTimestamp ifNil: [0]).
 	isActivationString := (anEvent keyString = self class activationString).
```

Comment:
```suggestion
	isActivationString := (anEvent keyString = self class activationString) and: anEvent isKeyUp.
```

On my machine already triggers with a single key press otherwise (keydown & keyup event)